app/pages/adduser/contextCreatorErrorHelper.test.js

```diff
@@ -0,0 +1,61 @@
+import { formatErrors, formatAllErrors, addErrorsToManifest } from './contextCreatorErrorHelper';
+import { errorMessages } from './errorStrings';
+import manifest from './manifest.json';
+
+const errors = [
+  { field: 'emailAddress', id: 'EmailRequired' },
+  { field: 'emailAddress', id: 'EmailTooLong' },
+  { field: 'lastName', id: 'LastNameTooLong' },
+];
+
+describe('contextCreatorErrorHelper', () => {
+  describe('formatErrors', () => {
+    it('should create error object with field as key and array of messages for one error', () => {
+      const formattedErrors = formatErrors([errors[0]]);
+      expect(formattedErrors).toEqual({ [errors[0].field]: [errorMessages[errors[0].id]] });
```

Comment:
I would agai just perhaps hardcode the expected value. Just so it's bit more clear what the expectation is. The data we pass in is static and within this file so shouldn't make the test brittle.